modules/tinymce/src/themes/silver/main/ts/ui/menus/contextmenu/platform/MobileContextMenu.ts

```diff
@@ -101,28 +91,47 @@ const setupiOSOverrides = (editor: Editor) => {
   };
 };
 
-const show = (editor: Editor, e: EditorEvent<TouchEvent>, items: MenuItems, backstage: UiFactoryBackstage, contextmenu: AlloyComponent, useNodeAnchor: boolean, highlightImmediately: boolean) => {
-  const anchorSpec = useNodeAnchor ? getNodeAnchor(editor) : getPointAnchorSpec(editor, e);
+const getAnchorSpec = (editor: Editor, e: EditorEvent<TouchEvent>, anchorType: Coords.ContextMenuAnchorType) => {
+  if (anchorType === 'point') {
+    return {
+      bubble: Bubble.nu(0, bubbleSize, bubbleAlignments),
+      layouts,
+      overrides: {
+        maxWidthFunction: MaxWidth.expandable(),
+        maxHeightFunction: MaxHeight.expandable()
+      },
+      ...Coords.getPointAnchor(editor, e)
+    };
+  } else {
+    return Coords.getAnchorSpec(editor, e, anchorType);
```

Comment:
Again, there's duplication here which we don't want... please use the example I gave in slack if you're taking this approach.

- [ ] Fix this to use the example given on slack whereby the `getAnchorSpec` call is not duplicated

Reply:
I thought it would be better than calling the method when is not needed. But if it's how we do it, I'm gonna change 👍

Reply:
Actually they are different: `getPointAnchor` and `getAnchorSpec`.

Reply:
No, they are the same as the intention is to lookup the anchor based on it's type and the augment it. So this ends up duplicating the lookup logic which we don't want.

Reply:
I should add this is still calling the same logic (so it's not calling anything that's unneeded), it's just you're manually short circuiting it, which doesn't convey at the core it should be the same just with some additional defaults.

Reply:
Calling `getPointAnchor` avoids the switch, that's the difference. Changed anyways.